test_container/tests/test/pandas/all/dataframe_memory_leak.py

```diff
@@ -111,6 +112,7 @@ def run(ctx):
                     assert memory_check_executed == False #Sanity check for row number
                     print("Checking memory usage", flush=True)
                     gc.collect()
+                    time.sleep(1)
```

Comment:
In case it works we should a comment to the sleeps

Reply:
Done. 
The CI Build failed because of Docker.